shell-completion/bash/portablectl

```diff
@@ -91,7 +91,7 @@ _portablectl() {
     elif __contains_word "$verb" ${VERBS[STANDALONE]}; then
         comps=''
     elif __contains_word "$verb" ${VERBS[IMAGE]}; then
-        if [[ $n == 1 ]]; then
+        if [[ $n == 1 ]] || [[ "$prev" = -* ]] || __contains_word "${COMP_WORDS[COMP_CWORD-2]}" ${OPTS[ARG]}; then
```

Comment:
COMP_CWORD is larger or equals to 2 ??

Reply:
you can chain many options so it goes up

Reply:
I mean, is it not necessary to check underflow?